packages/builders/src/messages/formatters.ts

```diff
@@ -182,6 +182,75 @@ export function roleMention<C extends Snowflake>(roleId: C): `<@&${C}>` {
 	return `<@&${roleId}>`;
 }
 
+/**
+ * Formats an application command name, subcommand group name, subcommand name, and ID into an application command mention
+ *
+ * @param commandName - The application command name to format
+ * @param subcommandGroupName - The subcommand group name to format
+ * @param subcommandName - The subcommand name to format
+ * @param commandId - The application command ID to format
+ */
+export function applicationCommandMention<N extends string, G extends string, S extends string, I extends Snowflake>(
+	commandName: N,
+	subcommandGroupName: G,
+	subcommandName: S,
+	commandId: I,
+): `</${N} ${G} ${S}:${I}>`;
```

Comment:
You shouldn't add this imo. It doesn't actually work, it just displays the text nicely but you can't click it.

Reply:
It works on my end